lib/api/BackbeatServer.js

```diff
@@ -104,6 +105,52 @@ class BackbeatServer {
         this._response(err, backbeatRequest);
     }
 
+    /**
+     * Get the body of a POST request and route it accordingly.
+     * @param {ClientRequest} req - The incoming request
+     * @param {BackbeatRequest} bbRequest - The Backbeat API Request
+     * @param {Object} routeDetails - The Backbeat router route details
```

Comment:
"router route", do you mean "Backbeat route"?

Reply:
Updated to "Backbeat route".